addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -169,6 +255,8 @@ def _check_leave_type_validity(self):
     @api.model
     def create(self, values):
         """ Override to avoid automatic logging of creation """
+        if values.get('accrual', False):
```

Comment:
So, we can not create an allocation starting in the past, or in the future ? If you should be able to do so, I think forcing the field required when 'accrual' is check on form view might be good, and maybe a constratint too.